data-manipulation/encryption/speck/encrypt-data-using-speck.yml

```diff
@@ -0,0 +1,37 @@
+rule:
+  meta:
+    name: encrypt data using speck
+    namespace: data-manipulation/encryption/speck
+    authors:
+      - still@teamt5.org
+    scopes:
+      static: function
```

Comment:
Can this (or a subset of features) be tightened with _basic block_ to avoid potential FPs?

Reply:
I've experimented a bit, and I'm not entirely sure how I'd go about matching this at block level.